core/src/main/kotlin/com/katanox/tabour/Tabour.kt

```diff
@@ -42,6 +42,12 @@ class Tabour internal constructor(val config: Configuration) {
      */
     fun <T> register(registry: Registry<T>): Tabour = this.apply { registries.add(registry) }
 
+    fun updateRegistries(registries: Iterable<Registry<*>>) =
```

Comment:
General question, 

Was there a case that we a have a registry not added in the `registries` collection?, that's why we created the updateRegistries?

Reply:
no this was to change the registries during runtime because of spring, but i changed the mechanism and removed the update